index.js

```diff
@@ -91,12 +92,13 @@ client.on("message", async (message) => {
 	}
 
 	// stops banned players
-	if (userProfile.account.banned) {
-		return message.reply("You are banned from Mega-RPG. You can plead for an unban at our support servers");
+	if (userProfile.account.banTime > Date.now()) {
+		console.log(userProfile.account.banTime, typeof userProfile.account.banTime);
+		return message.reply(`You are banned from Mega-RPG. You can plead for an unban at our support servers - or wait:\n **${msToHumanTime(userProfile.account.banTime - Date.now())}**`);
 	}
 
-	if (Math.random() <= 0.02 && userProfile.account.testUser === false && ["hunt", "collect", "raid", "fish"].includes(command.name)) {
-		return handleCaptcha(message, userProfile, 3);
+	if (Math.random() <= 1.02 && userProfile.account.testUser === false && ["hunt", "collect", "raid", "fish"].includes(command.name)) {
```

Comment:
1) userProfile.account.testUser === false: Kan vel skiftes til !userProfile.account.testUser
2) Er det ikke bedre å teste for ["hunt", "collect", "raid", "fish"].includes(command.name) først? Siden jeg tror Math.random() er en mer krevende funksjon?
3) Vil ikke Math.random() alltid være mindre enn 1?

Reply:
done